website/content/docs/platform/k8s/vso/auth/aws.mdx

```diff
@@ -0,0 +1,222 @@
+---
+layout: docs
+page_title: Vault Secrets Operator AWS Auth
+description: >-
+  The Vault Secrets Operator supports authenticating to Vault with the AWS auth method
+---
+
+# AWS auth method support
+
+The Vault Secrets Operator (VSO) supports authenticating to Vault's [AWS auth](/vault/docs/auth/aws) method. AWS credentials can be retrieved in several ways:
+
+- From an [IRSA-enabled Kubernetes service account][aws-irsa]
+- Inferred from the underlying EKS node role or EC2 instance profile of the instance where the operator pod is running
```

Comment:
```suggestion
- by inferring credentials from the underlying EKS node role.
- by inferring credentials from the EC2 instance profile of the instance
   where the operator pod is running.
```
Style correction: write in the active voice, use one idea per sentence. The instance profile example is particularly hard for the reader to parse. Moving that idea to its own line helps with understanding.